README.adoc

```diff
@@ -4,7 +4,10 @@
 :toc: left
```

Comment:
I've taken the liberty to shrink the README, and moved all development related stuff to CONTRIBUTING
Created a Development section and moved it right below Introduction.
My line of reasoning was: If I want to contribute, I do want a short Introduction to the project, and then I just want to know: "How to I build this?"